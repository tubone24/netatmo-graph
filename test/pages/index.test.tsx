import React from 'react'
import { render } from '../testUtils'
import Home from '../../pages/index'
import { RecoilRoot } from 'recoil'

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <RecoilRoot>
        <Home />
      </RecoilRoot>,
      {}
    )
    // https://github.com/mui-org/material-ui/issues/21293#issuecomment-654921524
    expect(
      asFragment()
        .firstElementChild.innerHTML.replace(/id="mui-[0-9]*"/g, '')
        .replace(/aria-labelledby="(mui-[0-9]* *)*"/g, '')
    ).toMatchSnapshot()
  })
})
