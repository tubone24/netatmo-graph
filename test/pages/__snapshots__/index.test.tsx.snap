// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `"<main class=\\"jsx-531684522\\"><div class=\\"MuiPaper-root MuiPaper-elevation2 MuiPaper-rounded\\" style=\\"position: relative;\\"><div class=\\"MuiToolbar-root MuiToolbar-regular MTableToolbar-root-5 MuiToolbar-gutters\\"><div class=\\"MTableToolbar-title-9\\"><div class=\\"jsx-531684522 header\\"><img src=\\"/awslogo.png\\" class=\\"jsx-531684522\\"><a href=\\"https://aws-health-dashboard.vercel.app/\\" class=\\"jsx-531684522\\">AWS Health Dashboard</a></div></div><div class=\\"MTableToolbar-spacer-7\\"></div><div class=\\"MuiFormControl-root MuiTextField-root MTableToolbar-searchField-10\\"><div class=\\"MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedStart MuiInputBase-adornedEnd\\"><div class=\\"MuiInputAdornment-root MuiInputAdornment-positionStart\\"><svg class=\\"MuiSvgIcon-root MuiSvgIcon-fontSizeSmall\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" title=\\"Search\\"><path d=\\"M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z\\"></path></svg></div><input aria-invalid=\\"false\\" placeholder=\\"Search\\" type=\\"text\\" aria-label=\\"Search\\" class=\\"MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedStart MuiInputBase-inputAdornedEnd\\" value=\\"\\"><div class=\\"MuiInputAdornment-root MuiInputAdornment-positionEnd\\"><button class=\\"MuiButtonBase-root MuiIconButton-root Mui-disabled Mui-disabled\\" tabindex=\\"-1\\" type=\\"button\\" disabled=\\"\\" aria-label=\\"Clear Search\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root MuiSvgIcon-fontSizeSmall\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" aria-label=\\"clear\\"><path d=\\"M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z\\"></path></svg></span></button></div></div></div><div class=\\"MTableToolbar-actions-8\\"><div><div><span><button class=\\"MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit\\" tabindex=\\"0\\" type=\\"button\\" aria-label=\\"Export\\" title=\\"Export\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M19 12v7H5v-7H3v7c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2v-7h-2zm-6 .67l2.59-2.58L17 11.5l-5 5-5-5 1.41-1.41L11 12.67V3h2z\\"></path></svg></span><span class=\\"MuiTouchRipple-root\\"></span></button></span><span><button class=\\"MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit\\" tabindex=\\"0\\" type=\\"button\\" title=\\"Show Bar Chart\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M5 9.2h3V19H5zM10.6 5h2.8v14h-2.8zm5.6 8H19v6h-2.8z\\"></path></svg></span><span class=\\"MuiTouchRipple-root\\"></span></button><button class=\\"MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit\\" tabindex=\\"0\\" type=\\"button\\" title=\\"Refresh Data\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z\\"></path></svg></span><span class=\\"MuiTouchRipple-root\\"></span></button></span></div></div></div></div><div class=\\"MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters\\" style=\\"padding: 0px; min-height: unset;\\"><div style=\\"background: rgba(0, 0, 0, 0.039); display: flex; width: 100%; padding: 8px; overflow: auto; border: 1px solid #ccc; border-style: dashed;\\"><span class=\\"MuiTypography-root MuiTypography-caption\\" style=\\"padding: 8px;\\">Drag headers here to group by</span></div></div><div class=\\"Component-horizontalScrollContainer-12\\" style=\\"overflow-x: auto; position: relative;\\"><div><div style=\\"overflow-y: auto;\\"><div><table class=\\"MuiTable-root\\" style=\\"table-layout: auto;\\"><thead class=\\"MuiTableHead-root\\"><tr class=\\"MuiTableRow-root MuiTableRow-head\\"><th class=\\"MuiTableCell-root MuiTableCell-head MTableHeader-header-13 MuiTableCell-paddingNone\\" scope=\\"col\\" style=\\"background-color: rgb(231, 127, 47); color: rgb(255, 255, 255);\\"></th><th class=\\"MuiTableCell-root MuiTableCell-head MTableHeader-header-13 MuiTableCell-alignLeft\\" scope=\\"col\\" style=\\"background-color: rgb(231, 127, 47); color: rgb(255, 255, 255); box-sizing: border-box;\\"><span class=\\"MuiButtonBase-root MuiTableSortLabel-root\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\"><div data-rbd-draggable-context-id=\\"0\\" data-rbd-draggable-id=\\"0\\" tabindex=\\"0\\" role=\\"button\\" aria-describedby=\\"rbd-hidden-text-0-hidden-text-0\\" data-rbd-drag-handle-draggable-id=\\"0\\" data-rbd-drag-handle-context-id=\\"0\\" draggable=\\"false\\">Service Name</div><svg class=\\"MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionDesc\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z\\"></path></svg></span></th><th class=\\"MuiTableCell-root MuiTableCell-head MTableHeader-header-13 MuiTableCell-alignLeft\\" scope=\\"col\\" style=\\"background-color: rgb(231, 127, 47); color: rgb(255, 255, 255); box-sizing: border-box;\\"><span class=\\"MuiButtonBase-root MuiTableSortLabel-root\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\"><div data-rbd-draggable-context-id=\\"0\\" data-rbd-draggable-id=\\"1\\" tabindex=\\"0\\" role=\\"button\\" aria-describedby=\\"rbd-hidden-text-0-hidden-text-0\\" data-rbd-drag-handle-draggable-id=\\"1\\" data-rbd-drag-handle-context-id=\\"0\\" draggable=\\"false\\">Service</div><svg class=\\"MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionDesc\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z\\"></path></svg></span></th><th class=\\"MuiTableCell-root MuiTableCell-head MTableHeader-header-13 MuiTableCell-alignLeft\\" scope=\\"col\\" style=\\"background-color: rgb(231, 127, 47); color: rgb(255, 255, 255); box-sizing: border-box;\\"><span class=\\"MuiButtonBase-root MuiTableSortLabel-root\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\"><div data-rbd-draggable-context-id=\\"0\\" data-rbd-draggable-id=\\"2\\" tabindex=\\"0\\" role=\\"button\\" aria-describedby=\\"rbd-hidden-text-0-hidden-text-0\\" data-rbd-drag-handle-draggable-id=\\"2\\" data-rbd-drag-handle-context-id=\\"0\\" draggable=\\"false\\">Region</div><svg class=\\"MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionDesc\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z\\"></path></svg></span></th><th class=\\"MuiTableCell-root MuiTableCell-head MTableHeader-header-13 MuiTableCell-alignLeft\\" scope=\\"col\\" style=\\"background-color: rgb(231, 127, 47); color: rgb(255, 255, 255); box-sizing: border-box;\\"><span class=\\"MuiButtonBase-root MuiTableSortLabel-root\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\"><div data-rbd-draggable-context-id=\\"0\\" data-rbd-draggable-id=\\"3\\" tabindex=\\"0\\" role=\\"button\\" aria-describedby=\\"rbd-hidden-text-0-hidden-text-0\\" data-rbd-drag-handle-draggable-id=\\"3\\" data-rbd-drag-handle-context-id=\\"0\\" draggable=\\"false\\">Summary</div><svg class=\\"MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionDesc\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z\\"></path></svg></span></th><th class=\\"MuiTableCell-root MuiTableCell-head MTableHeader-header-13 MuiTableCell-alignLeft\\" scope=\\"col\\" style=\\"background-color: rgb(231, 127, 47); color: rgb(255, 255, 255); box-sizing: border-box;\\"><span class=\\"MuiButtonBase-root MuiTableSortLabel-root MuiTableSortLabel-active\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\"><div data-rbd-draggable-context-id=\\"0\\" data-rbd-draggable-id=\\"4\\" tabindex=\\"0\\" role=\\"button\\" aria-describedby=\\"rbd-hidden-text-0-hidden-text-0\\" data-rbd-drag-handle-draggable-id=\\"4\\" data-rbd-drag-handle-context-id=\\"0\\" draggable=\\"false\\">Date (Asia/Tokyo)</div><svg class=\\"MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionDesc\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z\\"></path></svg></span></th><th class=\\"MuiTableCell-root MuiTableCell-head MTableHeader-header-13 MuiTableCell-alignLeft\\" scope=\\"col\\" style=\\"background-color: rgb(231, 127, 47); color: rgb(255, 255, 255); box-sizing: border-box;\\"><span class=\\"MuiButtonBase-root MuiTableSortLabel-root\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\"><div data-rbd-draggable-context-id=\\"0\\" data-rbd-draggable-id=\\"5\\" tabindex=\\"0\\" role=\\"button\\" aria-describedby=\\"rbd-hidden-text-0-hidden-text-0\\" data-rbd-drag-handle-draggable-id=\\"5\\" data-rbd-drag-handle-context-id=\\"0\\" draggable=\\"false\\">Status</div><svg class=\\"MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionDesc\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z\\"></path></svg></span></th></tr></thead><tbody class=\\"MuiTableBody-root\\"><tr class=\\"MuiTableRow-root\\" style=\\"height: 10px;\\"><td class=\\"MuiTableCell-root MuiTableCell-body MuiTableCell-paddingNone\\"></td><td class=\\"MuiTableCell-root MuiTableCell-body\\"><div class=\\"MuiFormControl-root MuiTextField-root\\"><div class=\\"MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedStart\\"><div class=\\"MuiInputAdornment-root MuiInputAdornment-positionStart\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" title=\\"Filter\\"><path d=\\"M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z\\"></path></svg></div><input aria-invalid=\\"false\\" placeholder=\\"\\" type=\\"search\\" aria-label=\\"filter data by Service Name\\" class=\\"MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedStart MuiInputBase-inputTypeSearch MuiInput-inputTypeSearch\\" value=\\"\\"></div></div></td><td class=\\"MuiTableCell-root MuiTableCell-body\\"><div class=\\"MuiFormControl-root MuiTextField-root\\"><div class=\\"MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedStart\\"><div class=\\"MuiInputAdornment-root MuiInputAdornment-positionStart\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" title=\\"Filter\\"><path d=\\"M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z\\"></path></svg></div><input aria-invalid=\\"false\\" placeholder=\\"\\" type=\\"search\\" aria-label=\\"filter data by Service\\" class=\\"MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedStart MuiInputBase-inputTypeSearch MuiInput-inputTypeSearch\\" value=\\"\\"></div></div></td><td class=\\"MuiTableCell-root MuiTableCell-body\\"><div class=\\"MuiFormControl-root\\" style=\\"width: 100%;\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated\\" data-shrink=\\"false\\" for=\\"select-multiple-checkbox2\\" style=\\"margin-top: -16px;\\"></label><div class=\\"MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl\\" style=\\"margin-top: 0px;\\"><div class=\\"MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input\\" tabindex=\\"0\\" role=\\"button\\" aria-haspopup=\\"listbox\\"><span>​</span></div><input aria-hidden=\\"true\\" tabindex=\\"-1\\" class=\\"MuiSelect-nativeInput\\" id=\\"select-multiple-checkbox2\\" value=\\"\\"><svg class=\\"MuiSvgIcon-root MuiSelect-icon\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M7 10l5 5 5-5z\\"></path></svg></div></div></td><td class=\\"MuiTableCell-root MuiTableCell-body\\"><div class=\\"MuiFormControl-root MuiTextField-root\\"><div class=\\"MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedStart\\"><div class=\\"MuiInputAdornment-root MuiInputAdornment-positionStart\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" title=\\"Filter\\"><path d=\\"M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z\\"></path></svg></div><input aria-invalid=\\"false\\" placeholder=\\"\\" type=\\"search\\" aria-label=\\"filter data by Summary\\" class=\\"MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedStart MuiInputBase-inputTypeSearch MuiInput-inputTypeSearch\\" value=\\"\\"></div></div></td><td class=\\"MuiTableCell-root MuiTableCell-body\\"><div class=\\"MuiFormControl-root MuiTextField-root\\"><div class=\\"MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedStart\\"><div class=\\"MuiInputAdornment-root MuiInputAdornment-positionStart\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" title=\\"Filter\\"><path d=\\"M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z\\"></path></svg></div><input aria-invalid=\\"false\\" placeholder=\\"\\" type=\\"search\\" aria-label=\\"filter data by Date (Asia/Tokyo)\\" class=\\"MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedStart MuiInputBase-inputTypeSearch MuiInput-inputTypeSearch\\" value=\\"\\"></div></div></td><td class=\\"MuiTableCell-root MuiTableCell-body\\"><div class=\\"MuiFormControl-root\\" style=\\"width: 100%;\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated\\" data-shrink=\\"false\\" for=\\"select-multiple-checkbox5\\" style=\\"margin-top: -16px;\\"></label><div class=\\"MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl\\" style=\\"margin-top: 0px;\\"><div class=\\"MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input\\" tabindex=\\"0\\" role=\\"button\\" aria-haspopup=\\"listbox\\"><span>​</span></div><input aria-hidden=\\"true\\" tabindex=\\"-1\\" class=\\"MuiSelect-nativeInput\\" id=\\"select-multiple-checkbox5\\" value=\\"\\"><svg class=\\"MuiSvgIcon-root MuiSelect-icon\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M7 10l5 5 5-5z\\"></path></svg></div></div></td></tr><tr class=\\"MuiTableRow-root\\" index=\\"0\\" level=\\"0\\" path=\\"0\\" style=\\"transition: all ease 300ms;\\"><td class=\\"MuiTableCell-root MuiTableCell-body MuiTableCell-paddingNone\\"><div style=\\"width: 42px; text-align: center; display: flex;\\"><button class=\\"MuiButtonBase-root MuiIconButton-root\\" tabindex=\\"0\\" type=\\"button\\" style=\\"transition: all ease 200ms; transform: none;\\" title=\\"Details\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z\\"></path></svg></span><span class=\\"MuiTouchRipple-root\\"></span></button></div></td><td class=\\"MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft\\" value=\\"Auto Scaling (N. Virginia)\\" style=\\"box-sizing: border-box; font-family: inherit; font-weight: inherit;\\">Auto Scaling (N. Virginia)</td><td class=\\"MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft\\" value=\\"autoscaling\\" style=\\"box-sizing: border-box; font-family: inherit; font-weight: inherit;\\">autoscaling</td><td class=\\"MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft\\" value=\\"US East (N. Virginia)\\" style=\\"box-sizing: border-box; font-family: inherit; font-weight: inherit;\\">US East (N. Virginia)</td><td class=\\"MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft\\" value=\\"[RESOLVED] Example Error\\" style=\\"box-sizing: border-box; font-family: inherit; font-weight: inherit;\\">[RESOLVED] Example Error</td><td class=\\"MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft\\" value=\\"1542849575\\" style=\\"box-sizing: border-box; font-family: inherit; font-weight: inherit;\\"><div class=\\"jsx-531684522\\">2018-11-22T10:19:35+09:00Z</div></td><td class=\\"MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft\\" value=\\"Informational message\\" style=\\"box-sizing: border-box; font-family: inherit; font-weight: inherit;\\">Informational message</td></tr><tr class=\\"MuiTableRow-root\\" style=\\"height: 49px;\\"></tr><tr class=\\"MuiTableRow-root\\" style=\\"height: 49px;\\"></tr><tr class=\\"MuiTableRow-root\\" style=\\"height: 49px;\\"></tr><tr class=\\"MuiTableRow-root\\" style=\\"height: 49px;\\"></tr><tr class=\\"MuiTableRow-root\\" style=\\"height: 1px;\\"></tr></tbody></table></div></div></div></div><table class=\\"MuiTable-root\\"><tfoot class=\\"MuiTableFooter-root\\" style=\\"display: grid;\\"><tr class=\\"MuiTableRow-root MuiTableRow-footer\\"><td class=\\"MuiTableCell-root MuiTableCell-footer MuiTablePagination-root Component-paginationRoot-1\\" colspan=\\"3\\" style=\\"float: right; overflow-x: auto;\\" icons=\\"[object Object]\\"><div class=\\"MuiToolbar-root MuiToolbar-regular MuiTablePagination-toolbar Component-paginationToolbar-2 MuiToolbar-gutters\\"><div class=\\"MuiTablePagination-spacer\\"></div><p class=\\"MuiTypography-root MuiTablePagination-caption Component-paginationCaption-3 MuiTypography-body2 MuiTypography-colorInherit\\" >Rows per page:</p><div class=\\"MuiInputBase-root MuiTablePagination-input MuiTablePagination-selectRoot Component-paginationSelectRoot-4\\"><div class=\\"MuiSelect-root MuiSelect-select MuiTablePagination-select MuiSelect-selectMenu MuiInputBase-input\\" tabindex=\\"0\\" role=\\"button\\" aria-haspopup=\\"listbox\\"  ><div style=\\"padding: 0px 5px;\\">5 rows </div></div><input aria-hidden=\\"true\\" tabindex=\\"-1\\" class=\\"MuiSelect-nativeInput\\" value=\\"5\\"><svg class=\\"MuiSvgIcon-root MuiSelect-icon MuiTablePagination-selectIcon\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M7 10l5 5 5-5z\\"></path></svg></div><p class=\\"MuiTypography-root MuiTablePagination-caption Component-paginationCaption-3 MuiTypography-body2 MuiTypography-colorInherit\\">1-1 of 1</p><div class=\\"MTablePaginationInner-root-14\\"><span title=\\"First Page\\" class=\\"\\"><button class=\\"MuiButtonBase-root MuiIconButton-root Mui-disabled Mui-disabled\\" tabindex=\\"-1\\" type=\\"button\\" disabled=\\"\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M18.41 16.59L13.82 12l4.59-4.59L17 6l-6 6 6 6zM6 6h2v12H6z\\"></path></svg></span></button></span><span title=\\"Previous Page\\" class=\\"\\"><button class=\\"MuiButtonBase-root MuiIconButton-root Mui-disabled Mui-disabled\\" tabindex=\\"-1\\" type=\\"button\\" disabled=\\"\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z\\"></path></svg></span></button></span><span class=\\"MuiTypography-root MuiTypography-caption\\" style=\\"flex: 1; text-align: center; align-self: center; flex-basis: inherit;\\">1-1 of 1</span><span title=\\"Next Page\\" class=\\"\\"><button class=\\"MuiButtonBase-root MuiIconButton-root Mui-disabled Mui-disabled\\" tabindex=\\"-1\\" type=\\"button\\" disabled=\\"\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z\\"></path></svg></span></button></span><span title=\\"Last Page\\" class=\\"\\"><button class=\\"MuiButtonBase-root MuiIconButton-root Mui-disabled Mui-disabled\\" tabindex=\\"-1\\" type=\\"button\\" disabled=\\"\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M5.59 7.41L10.18 12l-4.59 4.59L7 18l6-6-6-6zM16 6h2v12h-2z\\"></path></svg></span></button></span></div></div></td></tr></tfoot></table></div></main>"`;
